packages/prisma/schema.prisma

```diff
@@ -282,7 +282,8 @@ model Booking {
   status              BookingStatus        @default(ACCEPTED)
   paid                Boolean              @default(false)
   payment             Payment[]
-  destinationCalendar DestinationCalendar?
+  destinationCalendar DestinationCalendar? @relation(fields: [destinationCalendarId], references: [id])
+  destinationCalendarId Int? 
```

Comment:
run `yarn prisma format` to format these. Also VS Code should do this with prettier.